pkg/updater/updater.go

```diff
@@ -78,7 +79,7 @@ type disk interface {
 // Bootstrap bootstraps the default version for the given package.
 func Bootstrap(ctx context.Context, pkg string) error {
 	rc := newNoopRemoteConfig()
-	u := newUpdater(rc, defaultRepositoriesPath, defaultLocksPath)
+	u := newUpdater(rc, defaultRepositoriesPath, defaultLocksPath, "")
```

Comment:
why propagate the registry override instead of reading the config only in the downloader?